release/build.xml

```diff
@@ -43,61 +43,32 @@
     <os family="unix"/>
   </condition>
 
-  <property name="appName" value="geonetwork"/>
-  <property name="version" value="3.11.0"/>
+  <!-- Project build properties -->
+  <property name="appName" value="GeoNetwork"/>
+  <property name="displayName" value="GeoNetwork opensource"/>
+  <property name="homepage" value="https://geonetwork-opensource.org"/>
+  <property name="version" value="3.10.4"/>
```

Comment:
This should be `3.11.0`, no?

Reply:
I don't know what the next release will be ;)
Either way, this value should get replaced by the .sh release scripts.